src/admin/settings/settings.scss

```diff
@@ -0,0 +1,118 @@
+.components{
+    float: left;
+    position: relative;
+    min-height: 1px;
+    padding-right: 15px;
+    padding-left: 15px;
+}
+.importexport{
+    float: left;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    min-height: 20px;
+}
+.form-group{
+    >.settings-box-content{
+       border-radius: 4px;
+       border: 1px solid #D4E1F9;
+       padding: 0;
+       @include linear-gradient(to bottom, #eff8fd 0%, #F6FBFF 6%, #F6FBFF 16%, #F3FBFE 16%, #F0F9FE 37%, #F0F8FB 39%, #E8F5FB 57%, #E8F2FB 67%, #dbedf7 100%);
+       >.settings-box-header{
+           border-radius: 2px;
+           font-weight: bold;
+           line-height: 20px;
+           font-size: 15px;
+           padding: 5px 10px;
+           color: #c2eafe;
+           background: #144573;
+       }
+       >.settings-form{
+            border: transparent;
+            border-left:1px solid #D4E1F9;
+            padding: 10px 0 0 10px;
+            >legend{
+                color: #224477;
+                font-weight: bold;
+                font-size: 18px;
+                margin-top: 10px;
+                margin-bottom: 10px;
+                font-family: inherit;
+                line-height: 1.1;
+            }
+       }
+       .form-content{
+           margin: 10px;
+           input{
+               width: auto;
+               float: inherit;
+           }
+       }
+    }
+    .desc-label{
+        @extend .components;
+        width: 33.33333333%;
```

Comment:
(1) This is Sass, instead of writing a repeating value just, you can write `width: (1/3)%;` -- which will product an unrounded number.
(2) Don't style the interface this much